sportsref/ncaaf/players.py

```diff
@@ -139,6 +137,13 @@ def passing(self):
         doc = self.getDoc()
         table = doc('#passing')
         df = sportsref.utils.parseTable(table)
+        if df.empty and table.length > 0:
```

Comment:
Sorry you had to go through this, but this stuff is fixed in the new utils file, as you seem to have discovered. Want to merge in the current master branch to get the newest decorators, utils, etc and then replace this with the original code without this edge case? Same with the other functions.